app/controllers/analytics_controller.rb

```diff
@@ -30,6 +30,11 @@ def trends
     render "analytics/trends"
   end
 
+  def show
```

Comment:
@Woonyong92 Need to add in the access control layer to prevent merchants from accessing other merchants page.See other show controllers